pkg/config/config.go

```diff
@@ -316,9 +324,15 @@ func merge(oldConfig, newConfig *apiv1.Config) *apiv1.Config {
 		mergedConfig.PropagateProjectLabels = newConfig.PropagateProjectLabels
 	}
 
-	if len(newConfig.ServiceLBAnnotations) == 0 {
+	if len(newConfig.AllowTrafficFromNamespace) > 0 && newConfig.AllowTrafficFromNamespace[0] == "" {
+		mergedConfig.AllowTrafficFromNamespace = nil
+	} else if len(newConfig.AllowTrafficFromNamespace) > 0 {
+		mergedConfig.AllowTrafficFromNamespace = newConfig.AllowTrafficFromNamespace
+	}
```

Comment:
nit:
```suggestion
	if len(newConfig.AllowTrafficFromNamespace) == 0 || newConfig.AllowTrafficFromNamespace[0] == "" {
		mergedConfig.AllowTrafficFromNamespace = nil
	} else {
		mergedConfig.AllowTrafficFromNamespace = newConfig.AllowTrafficFromNamespace
	}
```

Reply:
Are you sure @thedadams ? Look at the above logic for similar scenarios:
```
if len(newConfig.PropagateProjectAnnotations) > 0 && newConfig.PropagateProjectAnnotations[0] == "" {
```

you cannot set mergedConfig.property to nil if the newConfig.property is just length zero. That would cause values to get stomped on on subsequent runs of acorn install if all the original flags are not reprovided, which is not the desired behavior

Reply:
So our reset logic for slice type arguments is to pass the argument with a single empty value:
```
acorn install --propogate-project-annotation ""
```

Reply:
Yep. My mistake. Thanks for double-checking me here.